core/arch/arm/pta/pta_cyres.c

```diff
@@ -313,6 +313,28 @@ static TEE_Result get_seal_key(struct cyres_pta_sess_ctx *ctx,
 
 	return res;
 }
+static TEE_Result handle_get_private_key_size(
+		struct cyres_pta_sess_ctx *ctx,
```

Comment:
Should only be indented one level

Reply:
optee uses the linux kernel style. I have a vim plugin that formats my code automatically according to the linux style, and this is the indentation it chose. It passes checkpatch.

Reply:
That's what I find surprising.  Linux kernel style, and the other files in this directory just have one tab here.